takwimu/templates/takwimu/_includes/search/pagination.html

```diff
@@ -15,13 +15,13 @@
                 <ul class="pagination pagination-sm"> 
                     <li class="page-item" data-page="prev">
                         <a class="page-link mx-1">
-                            Prev 
+                            Previous 
                             <span class="sr-only">(current)</span>
                         </a>
                     </li>
                     <!--Here the JS Function Will Add the Rows -->
-                    <li class="page-item" data-page="next" id="prev">
-                        <a class="page-link mx-1 active"> Next
+                    <li class="page-item active" data-page="next" id="prev">
```

Comment:
What if it's only one page of results?

Reply:
@DavidLemayian 
It wont show the pagination, the pagination script handle that